_includes/sidebar.html

```diff
@@ -77,7 +77,7 @@
                                     {% for deeplevel in thirdlevel.thirdlevelitems %}
                                         {% if deeplevel.external_url %}
                                         <li><a href="{{deeplevel.external_url}}" target="_blank">{{deeplevel.title}}</a></li>
-                                        {% elsif page.url == deeplevel.url %}
+                                        {% elsif page.url == deeplevel.url or deeplevel.alternate_urls contains page.url %}
```

Comment:
Nice! I've been wanting this for a long time but didn't think of this approach. Thank you!

@mjibson, with this change, we can, in principle, do away with filtering via javascript, breaking content onto unique pages instead. I know that's the eventual plan with Hugo anyway, though, so should I wait on that?